src/styles/other/media-query.scss

```diff
@@ -0,0 +1,5 @@
+@media (max-width: 1060px) {
+  .product-list {
+    padding: 50px 40px;
+  }
+}
```

Comment:
```suggestion
```
Move to product-list.scss and remove this file

Reply:
fixed